src/Parser/MistweaverMonk/Constants.js

```diff
@@ -0,0 +1,43 @@
+// Mistweaver Monk constants
+// Spells
+export const EFFUSE_HEAL_ID = 116694;
+export const VIVIFY_HEAL_ID = 116670;
+export const RENEWING_MIST_HEAL_ID = 115151;
+export const SHEILUNS_GIFT_HEAL_ID = 205405;
+export const ENVELOPING_MIST_HEAL_ID = 124682;
+export const ESSENCE_FONT_HEAL_ID = 191837;
+export const ESSENCE_FONT_HOT_ID = 191840;
+export const UPLIFTING_TRANCE_SPELL_ID = 197206;
+export const THUNDER_FOCUS_TEA_CAST_ID = 116680;
+export const LIFE_COCOON_CAST_ID = 116849;
+export const REVIVAL_HEAL_ID = 115310;
+
+
+
+// Traits
+export const BLESSINGS_OF_YULON_HEAL_ID = 199665;
+export const CELESTIAL_BREATH_HEAL_ID = 199640;
+export const TENDRILS_OF_REVIVAL_SPELL_ID = 238058;
+export const DANCING_MISTS_SPELL_ID = 199573;
+
+// Talents
+export const CHI_BURST_HEAL_ID = ;
+
+
+// General Spells
+export const LEECH_SPELL_ID = 143924;
+export const ARCANE_TORRENT_SPELL_ID = 155145;
+export const ANCIENT_HEALING_POTION_HEAL_SPELL_ID = 188016;
+export const MARK_OF_THE_ANCIENT_PRIESTESS_HEAL_SPELL_ID = 228401;
+
+export const ABILITIES_AFFECTED_BY_HEALING_INCREASES = [
```

Comment:
I think this should be the only thing in this file while pointing to `SPELLS.*.id`